deploy/runtools/runtime_config.py

```diff
@@ -138,26 +155,47 @@ def array_to_lognames(values, prefix):
+        disable_asserts = "+disable-asserts" if (hostdebug_config.disable_synth_asserts) else ""
+        print_cycle_prefix = "+print-no-cycle-prefix" if not synthprint_config.cycle_prefix else ""
 
         # TODO supernode support
         dwarf_file_name = "+dwarf-file-name=" + all_bootbinaries[0] + "-dwarf"
 
-        # TODO: supernode support (tracefile, trace-select.. etc)
-        basecommand = f"""screen -S fsim{slotid} -d -m bash -c "script -f -c 'stty intr ^] && sudo ./{driver} +permissive $(sed \':a;N;$!ba;s/\\n/ /g\' {runtimeconf}) +slotid={slotid} +profile-interval={profile_interval} {zero_out_dram} {disable_asserts} {command_macs} {command_rootfses} {command_niclogs} {command_blkdev_logs}  {tracefile} +trace-select={trace_select} +trace-start={trace_start} +trace-end={trace_end} +trace-output-format={trace_output_format} {dwarf_file_name} +autocounter-readrate={autocounter_readrate} {autocounterfile} {command_dromajo} {print_cycle_prefix} +print-start={print_start} +print-end={print_end} {command_linklatencies} {command_netbws}  {command_shmemportnames} +permissive-off {command_bootbinaries} && stty intr ^c' uartlog"; sleep 1"""
-
-        return basecommand
-
+        screen_name = "fsim{}".format(slotid)
+        run_device_placement = "+slotid={}".format(slotid) if self.platform == "f1" else "+device_index={}".format(slotid)
+        # TODO: re-enable for vitis
+        #other = "+binary_file={}".format(self.xclbin) if self.platform == "vitis" else ""
 
+        # TODO: supernode support (tracefile, trace-select.. etc)
+        permissive_driver_args = []
+        permissive_driver_args += [f"$(sed \':a;N;$!ba;s/\\n/ /g\' {runtimeconf})"]
+        permissive_driver_args += [run_device_placement]
+        permissive_driver_args += [f"+profile-interval={profile_interval}"]
+        permissive_driver_args += [zero_out_dram]
+        permissive_driver_args += [disable_asserts]
+        permissive_driver_args += command_macs
+        permissive_driver_args += command_rootfses
+        permissive_driver_args += command_niclogs
+        permissive_driver_args += command_blkdev_logs
+        permissive_driver_args += [f"{tracefile}", f"+trace-select={tracerv_config.select}", f"+trace-start={tracerv_config.start}", f"+trace-end={tracerv_config.end}", f"+trace-output-format={tracerv_config.output_format}", dwarf_file_name]
+        permissive_driver_args += [f"+autocounter-readrate={autocounter_config.readrate}", autocounterfile]
+        permissive_driver_args += [command_dromajo]
+        permissive_driver_args += [print_cycle_prefix, f"+print-start={synthprint_config.start}", f"+print-end={synthprint_config.end}"]
+        permissive_driver_args += command_linklatencies
+        permissive_driver_args += command_netbws
+        permissive_driver_args += command_shmemportnames
+        driver_call = f"""sudo ./{driver} +permissive {" ".join(permissive_driver_args)} +permissive-off {" ".join(command_bootbinaries)}"""
```

Comment:
Not relevant to this PR (but to the Vitis work later). Do we need `sudo` for the driver?